labextension/.yarnrc.yml

```diff
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
```

Comment:
I think this PR is ready to go, but it would be helpful to have the insights from this conversation captured in inline comments, combined with a configuration reference link or so to this kind of file in order to help us maintain this long term.

I may come back soon and add a code suggestion -> apply it -> and merge.

For inspiration on a top level comment, look at for example pre-commit-config.yaml that introduces the file and helps the reader understand how to maintain things in it a better.

Reply:
This file is a standard configuration in all other Jupyter projects: [JupyterLab](https://github.com/jupyterlab/jupyterlab/blob/main/.yarnrc.yml), [Notebook](https://github.com/jupyter/notebook/blob/main/.yarnrc.yml), [ipywidgets](https://github.com/jupyter-widgets/ipywidgets/blob/main/.yarnrc.yml) and also in the [template of JupyterLab extensions](https://github.com/jupyterlab/extension-cookiecutter-ts/blob/4.0/%7B%7Bcookiecutter.python_name%7D%7D/.yarnrc.yml)